test/integration_tests/test_print_files.py

```diff
@@ -497,6 +497,114 @@ def test_P_TB_TBPOL1(sftp_client):
         expected='|test_caseref|Mr|Test|McTest|123 Fake Street|Duffryn||Newport|NPXXXX|P_TB_TBPOL1\n')
 
 
+def test_P_OR_I1(sftp_client):
+    # Given
+    ichhqe_messages, _ = build_test_messages(print_questionnaire_message_template, 1, 'P_OR_IX', 'P_OR_I1')
```

Comment:
Small comment, variable is called `ichhqe_messages` but nothing to do with `ICHHQE`

Reply:
good spot, I'll fix it.  Probably just a copy and paste job of the tests